src/hotspot/share/gc/z/zRootsIterator.hpp

```diff
@@ -35,21 +35,9 @@
 
 class ZRootsIteratorClosure;
 
-typedef OopStorage::ParState<true /* concurrent */, false /* is_const */> ZOopStorageIterator;
```

Comment:
If you want you could also remove #include "gc/shared/oopStorageParState.hpp" from this file (though it's implicitly included, so will not matter more than fewer includes)

Reply:
Good point, will fix!